dotcom-rendering/src/lib/getDataLinkName.ts

```diff
@@ -58,14 +58,34 @@ export type Group = `${number}` | `${number}+`;
  *
  * @see {JSX.IntrinsicAttributes}
  */
+
+type MediaType =
+	| 'podcast'
+	| 'loop-video'
+	| 'youtube-video'
+	| 'slideshow'
+	| 'avatar'
+	| 'crossword'
+	| 'picture'
+	| 'none';
```

Comment:
```suggestion
import type { CardImageType } from '../types/layout';

type MediaType = CardImageType | 'none';
```

tying it to `CardImageType` is probably a good idea? Although renaming that type to `CardMediaType` would be nice too (I actually have a PR on the go to do this - I'll resurrect it soon)